epub33/overview/index.html

```diff
@@ -81,7 +81,7 @@
 	<body>
 		<section id="abstract">
 			<p>This document provides a starting point for content authors and software developers wishing to understand
-				the EPUB® 3 specifications. It consists entirely of informative overview material that describes the
+				the EPUB 3 specifications. It consists entirely of informative overview material that describes the
```

Comment:
Is this okay to remove now? It was required in IDPF that the first instance of "EPUB" not used in a specification name have the registered trademark symbol, since IDPF registered it.

I'm neither for nor against keeping, but if we're dropping here all the other specifications are likely to have one of these, too.

Reply:
Is there an answer to this?

Reply:
I'll open a separate issue since no point discussing in a merged one.